### DISCUSSION TITLE
Geting an image from the camera

### DISCUSSION THREAD

**Discussion by MareckiAI** [pybricks/support#1125] [Q&A]
is it possible to download the entire image from the camera to some multidimensional board because in the documentation I found only information on how to get the color

**Comment by dlech** [comment]
Can you explain more about what you are trying to accomplish? Which camera are you using?

**Reply by MareckiAI** [reply]
I use a camera that is called Color and Distance Sensor in the documentation and is included in the lego boost set. I wants to use this camera so that the processed image from it is an input for artificial intelligence

**Reply by dlech** [reply]
It is not a camera that can take photographic images. It only has a single sensor that can be used to detect light intensity, color and reflected infrared light.

It sounds like a Raspberry Pi with a camera or on OpenMV camera might work instead.
